samples/apps/copilot-chat-app/webapi/README.md

```diff
@@ -25,7 +25,11 @@ Before you get started, make sure you have the following requirements in place:
 
    > To clean your system of the developer certificate, run `dotnet run dev-certs https --clean`
 
-4. **(Optional)** [Visual Studio Code](http://aka.ms/vscode) or [Visual Studio](http://aka.ms/vsdownload).
+4. **(Optional)** To enable support for uploading image file formats such as png, jpg and tiff, we have included the [Tesseract](https://www.nuget.org/packages/Tesseract) nuget package.  
+   - You will need to obtain one or more [tessdata language data files](https://github.com/tesseract-ocr/tessdata) such as `eng.traineddata` and add them to your `./tessdata` directory.  
+   - Set the `Copy to Output Directory` value to `Copy if newer`.
```

Comment:
Please be more explicit on this step, i.e. where to set the value?

Reply:
+1, can we clarify this in `appsettings.json` as well?

Reply:
This clarification should exist in appsettings.json already.  Line 153 has some comments pertaining to this:
  //
  // Tesseract OCR Support (In order to upload image file formats such as png, jpg and tiff)
  // - Obtain language data files here: https://github.com/tesseract-ocr/tessdata . 
  // - Add these files to your `tessdata` folder and set the "Copy to Output Directory" value to "Copy if newer".
  //
  "Tesseract": {
    "Language": "eng"
  },

The primary README.md in the webapi project as well as a README.md that sits within the "tessdata" folder at the root of the the webapi project gives instructions on where to put the ".traineddata" files.  Happy to adjust verbiage if you all believe there is a better way to be more explicit here.

Reply:
I think I know where the confusion lies. [upon searching](https://stackoverflow.com/questions/18743907/visual-studio-how-to-copy-to-output-directory-without-copying-the-folder-stru) it looks like this option/value is only given in Visual Studio. 
I think we can exclude this part (in `appsetings.json` too) and assume that the user has placed the files correctly.